infrastructure/helm-chart/charts/prerequisites/charts/beanstalkd/BUILD

```diff
@@ -0,0 +1,29 @@
+load("@rules_pkg//:pkg.bzl", "pkg_tar")
+load("@com_github_airyhq_bazel_tools//helm:helm.bzl", "helm_template_test")
+load("//tools/build:helm.bzl", "helm_push_version")
+
+
+filegroup(
+    name = "files",
+    srcs = glob([
+        "**/*.yaml",
+        "**/*.tpl",
+    ]),
+)
+
+pkg_tar(
+    name = "package",
+    srcs = [":files"],
+    extension = "tgz",
+    strip_prefix = "./",
+)
+
+helm_template_test(
+    name = "template",
+    chart = ":package",
+)
+
+helm_push_version(
+    chart = ":package",
+    version = "1.0"
```

Comment:
Just a suggestion, but you could also wrap all of these rules up in a macro that only depends on the version. I.e.: 
```
helm_package(version = "1.0")
```

Reply:
Thanks, good suggestion :+1: . As we have two rules - `helm_push` and `helm_push_version`, I assume we will make some more improvements here, so I'd leave this to some next iteration.